.tool-versions

```diff
@@ -1,2 +1,4 @@
-elixir 1.10.2
-erlang 22.3
\ No newline at end of file
+elixir 1.11.2
+erlang 23.1.4
+rust 1.46.0
```

Comment:
I don't think it needs rust

Reply:
yeah not yet, but we should rem the libsec C nif from specs as well and go for the rust

Reply:
okay now we do :)